bip-update-process.md

```diff
@@ -0,0 +1,645 @@
+the broader community time. Not only may someone point out relevant discussion topics that were missed in the author’s
+research, or that an idea is guaranteed to be rejected based on prior discussions, but describing an idea publicly also
+tests whether it is of interest to more people beside the author. After establishing that the idea may be of interest
+to the Bitcoin community, the author should work on drafting a BIP.
+
+It is recommended that complicated proposals be split into separate BIPs that each focus on a specific component of the
+overall proposal.
+
+### Progression through BIP Statuses
+
+As the first draft is taking shape, the author should present it to the [Bitcoin development mailing
+list](https://groups.google.com/g/bitcoindev). This gives the author a chance to collect initial feedback and address
+fundamental concerns. If the author wishes to work in public on the draft at this stage, it is recommended that they
+open a pull request against their personal fork of the BIPs repository instead of the main BIPs repository.
+
+The following sections refer to BIP Status Field values. The BIP Status Field is defined in the Header Preamble
+specification above.
+
+#### Preliminary
+
+After fleshing out the draft further and ensuring that it is of high quality and properly formatted, the author should
+open a pull request to the [BIPs repository](https://github.com/bitcoin/bips). The draft BIP must adhere to the
+formatting requirements specified below and named with a working title of the form "bip-title". The author must not
+self-assign a number to the BIP draft.
+
+BIPs that (1) adhere to the formatting requirements, (2) are on-topic, and (3) have materially progressed
+beyond the ideation phase, e.g. by generating substantial public discussion and commentary from diverse contributors, by
+independent Bitcoin projects working on adopting the proposal, or by the author working for an extended period of time
+towards improving the proposal based on community feedback, will be assigned a number by a BIP editor. The BIP editors
+should delay number assignment when they perceive a BIP draft being met with comprehensive lack of interest: number
+assignment facilitates the distributed discussion of ideas, but before a BIP garners some interest of the Bitcoin
+community, there is no need to refer to it by a number.
+
+BIP drafts are also not eligible for number assignment if they duplicate efforts, disregard formatting rules, are too
+unfocused or too broad, fail to provide proper motivation, fail to address backwards compatibility, or fail to specify
+the feature clearly and completely. Reviewers and BIP editors should provide guidance on how the draft may
+be improved to progress towards readiness. Pull requests that are proposing off-topic or unserious ideas, or have
+stopped to make progress may be closed.
+
+When the BIP draft is complete, a BIP editor will assign a BIP type, and merge it to the BIPs repository.
```

Comment:
No, number assignment was already covered two paragraphs earlier. I did mean that a type would be assigned if it doesn’t have one yet. However, now that the types have been clarified, I would hope that this would simple not be an open question at that point.

Changed:
```diff
-When the BIP draft is complete, a BIP editor will assign a BIP type, and merge it to the BIPs repository.
+When the BIP draft has been assigned a number and is complete, a BIP editor will merge it to the BIPs repository.
```